packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -269,151 +285,132 @@ storiesOf('ListView', module)
-    </ListView>
+  .add('onAction', args => (
+    <NavigationExample {...args} />
   ))
-  .add('selectionStyle: checkbox, onAction', () => (
-    <ListView width="250px" height={400} onSelectionChange={action('onSelectionChange')} selectionMode="multiple" selectionStyle="checkbox" items={[...Array(20).keys()].map(k => ({key: k, name: `Item ${k}`}))} onAction={action('onAction')}>
-      {item => <Item>{item.name}</Item>}
-    </ListView>
+  .add('onAction, disable folder selection', args => (
+    <NavigationExample disabledType="folder" disabledBehavior="selection" {...args} />
   ))
-  .add('with ActionBar', () => <ActionBarExample />)
-  .add('with emphasized ActionBar', () => <ActionBarExample isEmphasized />)
-  .add('thumbnails', () => (
-    <ListView width="250px" items={itemsWithThumbs}>
-      {
-          (item) => <Item textValue={item.title}><Image src={item.url} /><Text>{item.title}</Text><Text slot="description">JPG</Text></Item>
-        }
-    </ListView>
+  .add('onAction, disable folder selection, with row actions', args => (
+    <NavigationExample disabledType="folder" disabledBehavior="selection" showActions {...args} />
   ))
-  .add('overflowMode="truncate" (default)', () => (
-    <ListView width="250px" overflowMode="truncate">
-      <Item textValue="row 1">row 1 with a very very very very very long title</Item>
-      <Item textValue="row 2">
-        <Text>Text slot with a really really really long name</Text>
-        <Text slot="description">Description slot with a really really long name</Text>
-      </Item>
-    </ListView>
+  .add('onAction, disable files', args => (
+    <NavigationExample disabledType="file" {...args} />
   ))
-  .add('overflowMode="wrap"', () => (
-    <ListView width="250px" overflowMode="wrap">
-      <Item textValue="row 1">row 1 with a very very very very very long title</Item>
-      <Item textValue="row 2">
-        <Text>Text slot with a really really really long name</Text>
-        <Text slot="description">Description slot with a really really long name</Text>
-      </Item>
-    </ListView>
+  .add('onAction, disable files, with row actions', args => (
+    <NavigationExample disabledType="file" showActions {...args} />
   ));
 
 storiesOf('ListView/Drag and Drop', module)
-    .add(
-      'Drag out of list',
-      () => (
-        <Flex direction="row" wrap alignItems="center">
-          <input />
-          <Droppable />
-          <DragExample
-            dragHookOptions={{onDragStart: action('dragStart'), onDragEnd: action('dragEnd')}} />
-        </Flex>
-      )
+  .addParameters(parameters)
+  .add(
+    'Drag out of list',
+    args => (
+      <Flex direction="row" wrap alignItems="center">
+        <input />
+        <Droppable />
+        <DragExample
+          dragHookOptions={{onDragStart: action('dragStart'), onDragEnd: action('dragEnd')}}
+          listViewProps={{args}} />
+      </Flex>
     )
+  )
   .add(
     'Drag within list (Reorder)',
```

Comment:
I'm noticing the drag handler shows up after a drag. It is existing behavior.
<img width="78" alt="image" src="https://user-images.githubusercontent.com/309540/169424226-c663d222-6d7d-4fcb-8787-39161b48b02a.png">

Reply:
I think because it becomes keyboard focused (intentionally I believe), so that causes the drag handle to also appear.

Reply:
This is the only story I saw do this.  🤷

Reply:
I'd expect `drag between lists` to behave the same, right? It doesn't show the drag handler after a drag from one list to the other.